### ISSUE TITLE
pt_PT translations (LOOT application, Setup and maybe documentation)

### ISSUE THREAD

**Issue by ironmagician** [loot/loot#1434]
Greetings. I just did most of the `pt_PT` translations for LOOT (almost at 90% coverage, following the provided contribution guidelines), yet I need some context for some of these messages.

Here is a list of the messages that I would require some context (grouping messages that are similar in context):

```
- 'Redate Plugins'? / 'Redate' / 'Plugins were successfully redated.'
- 'metadata edits'
- 'Loads Archive'
- 'Bash Tags' / 'Hide Bash Tags'
- 'A content string is required'
- 'Search Cards' / 'Search cards' - note this appears twice with different capitalizations, may not be intended.
- 'Click %(menu_icon)s buttons to open menus.' - would like some examples of what '%(menu_icon)' can be named.
```

Also, there were a few words that I'm having a hard time translating. Some of these should be kept as-is (e.g.: Plugins), others I am just having a hard time finding a proper solution (will look further into this). I'm leaving them here just to keep track, so that if someone else grabs this file knows the challenges at hand:

```
- 'Plugins' (as-is)
- 'modders' (as-is)
- 'output' (as-is)
- 'Masterlist' - Not very hard to translate, but might be reviewing ahead. Currently using the portuguese term `Lista-mestre`.
- 'build'  - hard to translate differenciating 'version' and 'build'. Will probably remain as-is.
- 'Bash Tag' - 'Bash' will remains as-is, 'Tag' I have translated.
- 'Parsing' - Hard to translate without losing context. WIP
- 'Merging' - Hard to translate without losing context. WIP
```

There is no hurry to answer this. I believe I won't have time to look into this until the weekend.
I will then have a look into the setup translations and after all seems fine, activating the language for selection.

As a last note, I've been checking the documentation and noticed you offer only English. This seems fine for the development and contributing sections, but the Help section could really be useful for the end-user.

Would you guys find it interesting having the 'Help' section also translated? Is it something that could be easily implemented in the documentation web page? (this to be done after reviewing the masterlists, of course)

**Comment by Ortham** [comment]
Regarding message contexts:

- `'Redate Plugins'? / 'Redate' / 'Plugins were successfully redated.'` - these are related to plugin redating, which changes the (modification, IIRC) timestamps of plugins to match their load order, with the earliest-loading plugin having the oldest timestamp.
- `metadata edits` - this is substituted into the string `You have not yet applied or cancelled your X. Are you sure you want to quit?`, in place of `X`.
- `Loads Archive` - this is displayed as a tooltip when hovering over the icon that indicates if the plugin loads a `.bsa` or `.ba2` (depending on the game) archive.
- `Bash Tags` - this is the title of a tab in the metadata editor, for the section that lists Bash Tag metadata entries.
- `Hide Bash Tags` - this is the text of a filter in the sidebar.
- `A content string is required` - this is the error message displayed if you add a message with no content in the metadata editor.
- 'Search Cards' / 'Search cards' - the capitalised version is the tooltip for the search icon (the magnifying glass), and the other is the label of the text input displayed after clicking on the search icon.
- `Click %(menu_icon)s buttons to open menus.` - `%(menu_icon)s` is the menu icon (three dots arranged vertically).

As for translating the website, I'm not sure how to do this - the site is generated by Jekyll, which doesn't appear to have built-in support for multilingual sites, though a quick search found a few blog posts talking about how to do it. It might be easier to switch to something else which provides built-in support though (e.g. Hugo, though I've never used it). I'll look into it at some point.

**Comment by ironmagician** [comment]
I'm done with the translation. The translation file has some comments to explain some harder translations (mostly the words I've listed above), to ease compreension if someone else works on it. 

I've created the PR. Feel free to close this after confirming everything is correct.

Unrelated note:  We have a file for `bg` (Bulgarian), but never was set to active.
I believe it only needs the installer settings (and custom message) and adding it to loot_settings. 
Might want to find someone to translated that phrase and activate it.
